Useless options used for pylint in CI

```
************* Module /home/runner/work/ddnet/ddnet/.pylintrc
.pylintrc:1:0: R0022: Useless option value for '--disable', 'C0326' was removed from pylint, see https://github.com/PyCQA/pylint/pull/3577. (useless-option-value)
.pylintrc:1:0: R0022: Useless option value for '--disable', 'C0330' was removed from pylint, see https://github.com/PyCQA/pylint/pull/3571. (useless-option-value)
.pylintrc:1:0: R0022: Useless option value for '--disable', 'R0201' was moved to an optional extension, see https://pylint.pycqa.org/en/latest/whatsnew/2/2.14/summary.html#removed-checkers. (useless-option-value)
```

See https://github.com/ddnet/ddnet/actions/runs/3256847399/jobs/5347714955#step:11:10

--- a/.pylintrc
+++ b/.pylintrc
@@ -22,3 +22,4 @@ disable=
   R0915, # too-many-statements
   W0511, # fixme
   W0603, # global-statement
+  R0022, # useless-option-value

